test/integration/dockerregistryclient_test.go

```diff
@@ -63,7 +63,7 @@ func retryWhenUnreachable(t *testing.T, f func() error) error {
 }
 
 func TestRegistryClientConnect(t *testing.T) {
-	c := dockerregistry.NewClient()
+	c := dockerregistry.NewClient(10 * time.Second)
```

Comment:
Can I get some constants in the update?  This looks a little silly.  `DefaultTimeout` or something.  Even a separate constructor for the 90% who don't care.